packages/ui5-config/src/types.ts

```diff
@@ -35,5 +57,3 @@ export interface MiddlewareConfig {
         ignoreCertError?: boolean;
     };
 }
-// TypeDefs using TypeScript 4.1 feature will break consumer compilation < 4.1
-// export { NodeComment, Path } from '@sap-ux/yaml';
```

Comment:
We still need to prevent export of types that use 4.1 features to avoid breaking the consumption by tools-suite.